packages/web-app-external/src/App.vue

```diff
@@ -52,14 +68,98 @@ export default defineComponent({
     LoadingScreen
   },
   setup() {
+    const store = useStore()
+    const router = useRouter()
+    const currentRoute = useRoute()
+    const clientService = useClientService()
+    const { loadFileInfoByIdTask } = useLoadFileInfoById({ clientService })
     const appName = useRouteQuery('app')
     const applicationName = computed(() => queryItemAsString(unref(appName)))
+
+    const fileIdQueryItem = useRouteQuery('fileId')
+    const fileId = computed(() => {
+      return queryItemAsString(unref(fileIdQueryItem))
+    })
+
+    const driveAliasAndItem = useRouteParam('driveAliasAndItem')
+
+    const getMatchingSpace = (id): SpaceResource => {
```

Comment:
I would rename it to get(or resolve)MatchingSpaceByFileId to avoid naming clashes with our composable, because at first, I tought we could use the function from the composable